Description: Disable expand-check
 This reduce all the runtime of all the semodule operation quite drastically
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=720220

--- a/src/semanage.conf
+++ b/src/semanage.conf
@@ -36,3 +36,6 @@ module-store = direct
 # version is necessary.
 #policy-version = 19
 
+# expand-check check neverallow rules when executing all semanage commands.
+# Large penalty in time if you turn this on.
+expand-check=0
